portfolio/src/main/webapp/index.html

```diff
@@ -4,13 +4,22 @@
     <meta charset="UTF-8">
     <title>Kaye's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
+    <div id="header">
```

Comment:
You can consider using the semantic "header" tag [1] for this.

Some discussion about exactly this case on SO: https://stackoverflow.com/questions/19748487/semantic-markup-why-header-instead-of-div-id-header

[1] https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

Reply:
Weird that I knew <nav> existed but not <header>. Thanks!